openedx/features/partners/helpers.py

```diff
@@ -75,11 +73,11 @@ def get_course_description(course):
     return description
 
 
-def import_module_using_slug(partner_slug):
+def import_form_using_slug(partner_slug):
     try:
-        return import_module(PARTNERS_VIEW_FRMT.format(slug=partner_slug))
+        return import_module(PARTNERS_FORM_FRMT.format(slug=partner_slug))
```

Comment:
First of all this is Nice to have:

To your problem of limiting the dev to using a specific name of the form, maybe This can be of help:

list all classes in the module (I'm sure we can find a clean way to do this on stackoverflow)

check if any class is a subclass of django.forms.Form and use that class. Ideally there'll be only one hence lets use the first.

Reply:
Thank you for this suggestion. I thought about this a bit, and it seems that if we do this, we'll be putting a further restriction at the developers that the only overridden can be that which has `PartnerRegistrationForm` as it's parent. Which might not always be the case, for instance, if a future partner's requirements are so different that we may not inherit the generic form to being with. So for now, I think it makes sense to just leave it like this. Let me know what you think?

Reply:
Read my suggestion again 🗡

Reply:
"django.forms.Form"